src/index.js

```diff
@@ -26,6 +26,12 @@ export const RCComponent = ({
   const setToastbarPosition = useToastStore((state) => state.setPosition);
   setToastbarPosition(toastBarPosition);
 
+  const [messageToEdit, setMessageToEdit] = useState({});
```

Comment:
Let's not have any prop-drilling or global state that can affect rendering of components. As we are using zustand as our global state management can we please stick to it?